python_modules/dagster/dagster/_serdes/serdes.py

```diff
@@ -672,11 +672,17 @@ def constructor_param_names(self) -> Sequence[str]:
 
 class PydanticModelSerializer(ObjectSerializer[T_PydanticModel]):
     def object_as_mapping(self, value: T_PydanticModel) -> Mapping[str, Any]:
-        return value.__dict__
+        value_dict = value.__dict__
+
+        # respect aliases
+        return {
+            field.alias if field.alias else key: value_dict[key]
```

Comment:
nit: can you () to make precedence more legible `(field.alias if field.alias else key): value_dict[key]`
this tripped me up trying to read it at first

Reply:
👍